File: app/src/main/java/mattframpton/endclothing_mattframpton/Model/Product.java
package mattframpton.endclothing_mattframpton.Model;

import com.google.gson.annotations.Expose;

@SuppressWarnings("unused")
public class Product {

    @Expose
    private String id;
    @Expose
    private String image;
    @Expose
    private String name;
    @Expose
    private String price;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPrice() {
        return price;
    }

    public void setPrice(String price) {
        this.price = price;
    }

}

File: app/src/main/java/mattframpton/endclothing_mattframpton/model/Product.kt
package mattframpton.endclothing_mattframpton.model

import com.google.gson.annotations.Expose

class Product {

    @Expose
    var id: String? = null
    @Expose
    var image: String? = null
    @Expose
    var name: String? = null
    @Expose
    var price: String? = null

}
